Source/ee/Ee_SubSystem.cpp

```diff
@@ -450,7 +450,7 @@ uint32 CSubSystem::IOPortReadHandler(uint32 nAddress)
 		                         nAddress, m_EE.m_State.nPC);
 	}
 
-	if((nAddress == CINTC::INTC_STAT) || (nAddress == CGSHandler::GS_CSR))
+	if((nAddress == CINTC::INTC_STAT) || (nAddress == CGSHandler::GS_CSR) || (nAddress == CGSHandler::GS_CSR_ALT))
```

Comment:
This check is for idle loop skipping purposes. 

This demo doesn't seem to check `GS_CSR_ALT (0x12000400)` to wait a frame's end, so we should remove this additional check.